### PR TITLE
AWS Organizations

### PR DESCRIPTION
**Pull request by johnnywilkes**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-dc.paloaltonetworks.com/browse/EXPANDR-5776

## Description
Added Organization boto3 commands: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/organizations.html
- Added the ***aws-organization-list-roots*** command: Lists the roots that are defined in the current organization.
- Added the ***aws-organization-list-children*** command: Lists all of the organizational units (OUs) or accounts that are contained in the specified parent OU or root.
- Added the ***aws-organization-list-parents*** command: Lists the root or organizational units (OUs) that serve as the immediate parent of the specified child OU or account.
- Added the ***aws-organization-describe-ou*** command: Retrieves information about an organizational unit (OU).
- Added the ***ws-organization-describe-account*** command: Retrieves Organizations-related information about the specified account.
- Added the ***aws-organization-describe-organization*** command: Retrieves information about the organization that the user's account belongs to.

## Must have
- [ ] Tests
- [X] Documentation

### CODE REVIEW

**Comment by johnnywilkes** [comment]
@ShirleyDenkberg , can you please review?

**Comment by johnnywilkes** [comment]
@MichaelYochpaz , I believe these pre-commit errors have nothing to do with changes I made: https://github.com/demisto/content/actions/runs/6013070988/job/16309715934

Let me know if you have any questions/comments/concerns

File: Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.yml [hunk 1]
```diff
@@ -3768,7 +3768,114 @@ script:
...
+      type: String
+    - contextPath: AWS.Organization.Roots.Name
+      description: The Amazon Resource Name (ARN) of the root.
+      type: String
+  - arguments:
+    - description: The maximum number of results to return in a single call. Specify a value between 1 and 1000.
+      name: max_results
+    - description: The token for the next set of results.
+      name: next_token
+    - description: The unique identifier (ID) for the parent root or OU whose children you want to list.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.yml:3789]:
```suggestion
    - description: The unique identifier (ID) for the parent root or organizational unit (OU) whose children you want to list.
```

File: Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.yml [hunk 2]
```diff
@@ -3768,7 +3768,114 @@ script:
...
+      type: String
+    - contextPath: AWS.Organization.Children.Type
+      description: The type of this child entity.
+      type: String
+  - arguments:
+    - description: The maximum number of results to return in a single call. Specify a value between 1 and 1000.
+      name: max_results
+    - description: The token for the next set of results.
+      name: next_token
+    - description: The unique identifier (ID) of the OU or account whose parent containers you want to list. Don't specify a root.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.yml:3812]:
```suggestion
    - description: The unique identifier (ID) of the organizational unit (OU) or account whose parent containers you want to list. Don't specify a root.
```

File: Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.yml [hunk 3]
```diff
@@ -3768,7 +3768,114 @@ script:
...
+      name: child_id
+      required: true
+    description: Lists the root or organizational units (OUs) that serve as the immediate parent of the specified child OU or account.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.yml:3815]:
```suggestion
    description: Lists the root or organizational units (OUs) that serve as the immediate parent of the specified child organizational unit (OU) or account.
```

File: Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.yml [hunk 4]
```diff
@@ -3768,7 +3768,114 @@ script:
...
+      description: The unique identifier (ID) of the parent entity.
+      type: String
+    - contextPath: AWS.Organization.Parents.Type
+      description: The type of the parent entity.
+      type: String
+  - arguments:
+    - description: The unique identifier (ID) of the Amazon Web Services account that you want information about. You can get the ID from the ListAccounts or ListAccountsForParent operations.
+      name: account_id
+      required: true
+    description: Retrieves Organizations-related information about the specified account.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.yml:3828]:
```suggestion
    description: Retrieves organizations-related information about the specified account.
```

File: Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.yml [hunk 5]
```diff
@@ -3768,7 +3768,114 @@ script:
...
+      type: String
+  - arguments:
+    - description: The unique identifier (ID) of the organizational unit that you want details about. You can get the ID from the ListOrganizationalUnitsForParent operation.
+      name: ou_id
+      required: true
+    description: Retrieves information about an organizational unit (OU).
+    name: aws-organization-describe-ou
+    outputs:
+    - contextPath: AWS.Organization.OU.Id
+      description: The unique identifier (ID) associated with this OU.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.yml:3870]:
```suggestion
      description: The unique identifier (ID) associated with this organizational unit (OU).
```

File: Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.yml [hunk 6]
```diff
@@ -3768,7 +3768,114 @@ script:
...
+      type: String
+    - contextPath: AWS.Organization.OU.Arn
+      description: The Amazon Resource Name (ARN) of this OU.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.yml:3873]:
```suggestion
      description: The Amazon Resource Name (ARN) of this organizational unit (OU).
```

File: Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.yml [hunk 7]
```diff
@@ -3768,7 +3768,114 @@ script:
...
+      type: String
+    - contextPath: AWS.Organization.OU.Name
+      description: The friendly name of this OU.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.yml:3876]:
```suggestion
      description: The friendly name of this organizational unit (OU).
```

File: Packs/AWS-EC2/Integrations/AWS-EC2/README.md [hunk 8]
```diff
@@ -3173,3 +3173,320 @@ There is no context output for this command.
...
+
+`aws-organization-list-children`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| max_results | The maximum number of results to return in a single call. Specify a value between 1 and 1000. | Optional | 
+| next_token | The token for the next set of results. | Optional | 
+| parent_id | The unique identifier (ID) for the parent root or OU whose children you want to list. | Required |
```
Comment (ShirleyDenkberg) [inline Packs/AWS-EC2/Integrations/AWS-EC2/README.md:3246]:
```suggestion
| parent_id | The unique identifier (ID) for the parent root or organizational unit (OU) whose children you want to list. | Required | 
```

File: Packs/AWS-EC2/Integrations/AWS-EC2/README.md [hunk 9]
```diff
@@ -3173,3 +3173,320 @@ There is no context output for this command.
...
+
+`aws-organization-list-parents`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| max_results | The maximum number of results to return in a single call. Specify a value between 1 and 1000. | Optional | 
+| next_token | The token for the next set of results. | Optional | 
+| child_id | The unique identifier (ID) of the OU or account whose parent containers you want to list. Don't specify a root. | Required |
```
Comment (ShirleyDenkberg) [inline Packs/AWS-EC2/Integrations/AWS-EC2/README.md:3302]:
```suggestion
| child_id | The unique identifier (ID) of the organizational unit (OU) or account whose parent containers you want to list. Don't specify a root. | Required | 
```

File: Packs/AWS-EC2/Integrations/AWS-EC2/README.md [hunk 10]
```diff
@@ -3173,3 +3173,320 @@ There is no context output for this command.
...
+>### AWS Parents
+>|Id|Type|
+>|---|---|
+>| r-3333 | ROOT |
+
+
+### aws-organization-describe-account
+
+***
+Retrieves Organizations-related information about the specified account.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-EC2/Integrations/AWS-EC2/README.md:3338]:
```suggestion
Retrieves organization-related information about the specified account.
```

File: Packs/AWS-EC2/Integrations/AWS-EC2/README.md [hunk 11]
```diff
@@ -3173,3 +3173,320 @@ There is no context output for this command.
...
+| --- | --- | --- |
+| ou_id | The unique identifier (ID) of the organizational unit that you want details about. You can get the ID from the ListOrganizationalUnitsForParent operation. | Required | 
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| AWS.Organization.OU.Id | String | The unique identifier \(ID\) associated with this OU. | 
+| AWS.Organization.OU.Arn | String | The Amazon Resource Name \(ARN\) of this OU. | 
+| AWS.Organization.OU.Name | String | The friendly name of this OU. |
```
Comment (ShirleyDenkberg) [inline Packs/AWS-EC2/Integrations/AWS-EC2/README.md:3467]:
```suggestion
| AWS.Organization.OU.Id | String | The unique identifier \(ID\) associated with this organizational unit (OU). | 
| AWS.Organization.OU.Arn | String | The Amazon Resource Name \(ARN\) of this organizational unit (OU). | 
| AWS.Organization.OU.Name | String | The friendly name of this organizational unit (OU). | 
```

File: Packs/AWS-EC2/ReleaseNotes/1_2_37.md [hunk 12]
```diff
@@ -0,0 +1,12 @@
...
+#### Integrations
+
+##### AWS - EC2
+
+- Added the ***aws-organization-list-roots*** command: Lists the roots that are defined in the current organization.
+- Added the ***aws-organization-list-children*** command: Lists all of the organizational units (OUs) or accounts that are contained in the specified parent OU or root.
+- Added the ***aws-organization-list-parents*** command: Lists the root or organizational units (OUs) that serve as the immediate parent of the specified child OU or account.
+- Added the ***aws-organization-describe-ou*** command: Retrieves information about an organizational unit (OU).
+- Added the ***aws-organization-describe-account*** command: Retrieves Organizations-related information about the specified account.
+- Added the ***aws-organization-describe-organization*** command: Retrieves information about the organization that the user's account belongs to.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-EC2/ReleaseNotes/1_2_37.md:11]:
```suggestion
- Added the following commands:
   - ***aws-organization-list-roots***: Lists the roots that are defined in the current organization.
   - ***aws-organization-list-children***: Lists all of the organizational units (OUs) or accounts that are contained in the specified parent OU or root.
   - ***aws-organization-list-parents***: Lists the root or organizational units (OUs) that serve as the immediate parent of the specified child OU or account.
   - ***aws-organization-describe-ou***: Retrieves information about an organizational unit (OU).
   - ***aws-organization-describe-account***: Retrieves Organizations-related information about the specified account.
   - ***aws-organization-describe-organization***: Retrieves information about the organization that the user's account belongs to.
```

**Comment by ShirleyDenkberg** [comment]
@MichaelYochpaz Doc review completed.

**Pull request closed without merging**